build.gradle.kts

```diff
@@ -8,7 +8,35 @@ configurations {
 }
 
 dependencies {
+  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
+
+  // Spring boot dependencies
+  implementation("org.springframework.boot:spring-boot-starter-security")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("org.springframework.boot:spring-boot-starter-actuator")
```

Comment:
You don't need this - it's brought in by `uk.gov.justice.hmpps.gradle-spring-boot`